pipelines/models/ab-jq/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM alpine:3.18.3@sha256:7144f7bab3d4c2648d7e59409f15ec52a18006a128c733fcff20d3a4a54ba44a
+
+# `jq` is required to parse and format JSON
+# `apache2-utils` provides ab benchmarking tool
+# `bash` is required to allow stnadardized scripting in bash
+# `coreutils` is required for using functions like `date -d`
+# hadolint ignore=DL3018
+RUN apk update &&       \
+    apk add --no-cache jq apache2-utils bash coreutils &&  \
```

Comment:
Could probably use `ubi9` for this (alpine is fine though):
- `jq` [via appstream repo](https://cdn-ubi.redhat.com/content/public/ubi/dist/ubi9/9/x86_64/appstream/os/Packages/j/)
- `ab` is also provided by `httpd-tools` [via appstream repo](https://cdn-ubi.redhat.com/content/public/ubi/dist/ubi9/9/x86_64/appstream/os/Packages/h/)
- `bash` and `coreutils` preinstalled even in `ubi9/ubi-micro`

Reply:
+1 on the UBI usage. IMHO we should always use UBI.

Reply:
If there are missing packages in the ubi-image that require additional epel repos then centos base image is a suitable alternative.  We (ODH) had the same discussions where we needed to work around ubi limitations and it some instances it was easier to use centos-stream

Reply:
Changed to UBI image.